### ISSUE TITLE
[265] Request for AI Assistance Chat Bot in Documentation

### ISSUE THREAD

**Issue by near-dx** [near/docs#2258]
**Title:** 
Request for AI Assistance Chat Bot in Documentation
**Description:** 
we are an AI focused ecosystem but the docs not even have an AI assitance chat bot.
Take a look at other eco docs like SUI... they have a very useful AI chat bot
**Summary:** 
The customer is dissatisfied with the lack of an AI assistance chat bot in the documentation. They compare our ecosystem to others like SUI, which have such features. Address the customer's concerns by considering the implementation of an AI chat bot to enhance user experience.
## Info
**Reported by:** Guest [(randomeguy@example.com)](mailto:randomeguy@example.com) - [Open in Gleap](https://app.gleap.io/projects/6272ba4eb62acc0036624b99/bugs/crdL4WMS0pvazatjm1naKR78yuBUD2UTBVk34cz635G5H9OUwiMlwev9B3ANkoGRIqnmnG)
**Location:** VN
**Type:** 🚨 BUG

🔗 [Open share link](https://app.gleap.io/share/crdL4WMS0pvazatjm1naKR78yuBUD2UTBVk34cz635G5H9OUwiMlwev9B3ANkoGRIqnmnG?token=4GU6frF2qNVZ6xZVrtzHXtygHJHXyjEj0xS9qjw1JjKMssmDkLwV3Lb8AnyzmUXoVICtiq)
🌍 [Open ticket in Gleap](https://app.gleap.io/projects/6272ba4eb62acc0036624b99/bugs/crdL4WMS0pvazatjm1naKR78yuBUD2UTBVk34cz635G5H9OUwiMlwev9B3ANkoGRIqnmnG)
## Metadata
**browserName:** Chrome(129.0.0.0)
**userAgent:** Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
**browser:** Chrome
**systemName:** Linux
**sessionDuration:** 71
**devicePixelRatio:** 1.100000023841858
**screenWidth:** 1920
**screenHeight:** 1200
**innerWidth:** 1745
**innerHeight:** 985
**currentUrl:** https://docs.near.org/
**language:** en-US
**sdkVersion:** 13.7.3
**sdkType:** javascript
**environment:** prod

![Screenshot](https://staticfiles.gleap.io/p6272ba4eb62acc0036624b99/AS4vmjpceITYYCPrksTVNJnWs7NGaRXtRSkMyKn4SrIZclyv8wWzNvSp8mcOQgcK2srnkx0046h.jpg)

**Comment by near-dx** [comment]
Internal note 📝: The customer is dissatisfied with the lack of an AI assistance chat bot in the documentation. They compare our ecosystem to others like SUI, which have such features. Address the customer's concerns by considering the implementation of an AI chat bot to enhance user experience.

**Comment by bucanero** [comment]
@gagdiez I think this issue would be a duplicate of #2485 , as they request an AI chat bot too? 🤔

**Comment by gagdiez** [comment]
Indeed, let's leave it so we show that the issue had support from the community

**Comment by thisisjoshford** [comment]
Shipped :)
